tests/behat/features/pa/06-submit-report.feature

```diff
@@ -0,0 +1,52 @@
+Feature: Report submit
+
+    Scenario: report declaration page
+        Given I load the application status from "pa-report-completed"
+        And I am logged in as "behat-pa1@publicguardian.gsi.gov.uk" with password "Abcd1234"
+        And I click on "tab-ready"
+        And I click on "pa-report-open" in the "client-1000014" region
+        Then I should not see the "download-2016-report" link
+        # if not found, it means that the report is not submittable
+        And I click on "report-submit"
+        Then the URL should match "/report/\d+/review"
+        And I click on "declaration-page"
+        Then the URL should match "/report/\d+/declaration"
+        And I save the page as "report-submit-declaration"
```

Comment:
minor/not blocking: to avoid overriding debug pages, I'd prefix those with "pa", e.g. "pa-report-submit-declaration"